### PR TITLE
some touchups, while ramping up

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
+.PHONY: test
 test: fmt lint
 	npm run test
-.PHONY: test
+
+.PHONY: devenv
+devenv: .done/devenv
+
+# indirect through a .done file so that this only runs when deps have changed
+.done/devenv: sbin/install-deps Brewfile jsonnetfile.json package.json
```
Comment (mattgauntseo-sentry) [inline Makefile:19]: I'm not following the .done directory/rules here but it seems like an over complication for a simple "install dependencies" step.

File: jsonnetfile.json [hunk 2]
```diff
@@ -1,5 +1,15 @@
 {
   "version": 1,
-  "dependencies": [],
+  "dependencies": [
```
Comment (mattgauntseo-sentry) [inline jsonnetfile.json:3]: Why are you adding gocd-jsonnet as a dependency of itself?

**Review by mattgauntseo-sentry — CHANGES_REQUESTED** [review]

File: jsonnetfile.json (hunk 2 above)
Comment (bukzor-sentryio) [reply jsonnetfile.json:3]: I was just following the readme:

```
jb install https://github.com/getsentry/gocd-jsonnet.git/libs@v1.4.2
```

It does seem wrong though.

Comment (mattgauntseo-sentry) [reply jsonnetfile.json:3]: The README is geared to "This is how you use this library", not "This is everything you need to do to build this library"

Comment (mattgauntseo-sentry) [reply jsonnetfile.json:3]: You could add a jsonnetfile.json to the example if you wanted to have an example project that builds from start to finish?

**Review by mattgauntseo-sentry — APPROVED** [review]

File: vendor/github.com/getsentry/.gitignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# ... except for our self-symlink
```
Comment (mattgauntseo-sentry) [inline vendor/github.com/getsentry/.gitignore:1]: I'd add vendor/ to the .gitignore

**Review by mattgauntseo-sentry — APPROVED** [review]

**Review by mattgauntseo-sentry — APPROVED** [review]

File: vendor/github.com/getsentry/.gitignore (hunk 3 above)
Comment (bukzor-sentryio) [reply vendor/github.com/getsentry/.gitignore:1]: I have vendor/.gitignore specify `*`, equivalently.

**Comment by mwarkentin** [comment]
Seems stale?

**Pull request closed without merging**